shared/models.go

```diff
@@ -69,7 +69,9 @@ type TestRun struct {
 	// Results URL
 	ResultsURL string `json:"results_url"`
 
-	CreatedAt time.Time `json:"created_at"`
+	CreatedAt     time.Time `json:"created_at"`
+	RevisionHash  string    `json:"revision_hash"`
```

Comment:
This belongs in `ProductAtRevision` above (since it'll be wanted whenever `Revision` is wanted).

Also, let's use `FullRevisionHash` to be nice and explicit about the difference between the fields.